<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dinein_order</class>
 <widget class="QWidget" name="Dinein_order">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>请点餐</string>
  </property>
  <widget class="QPushButton" name="nameselect_button">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>名称查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>320</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;当前订单：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="typeselect_button">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>分类查询</string>
   </property>
  </widget>
  <widget class="QTableView" name="dinein_menu">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>650</width>
     <height>541</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="order_now">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>350</y>
     <width>650</width>
     <height>581</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="selectline">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="sale_today">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>40</y>
     <width>631</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;今日折扣：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Update_button">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>920</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Delete_button">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>850</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Demand">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>940</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>890</y>
     <width>381</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>有任何忌口或对这道菜的要求吗？</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Insert_button">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>780</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>610</y>
     <width>411</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;编号：     &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="dinein_dishid"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;名称： &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="dinein_dishname"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;分类： &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="dinein_dishtype"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;价格： &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="dinein_dishprice"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="dinein_dishnum">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>870</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>870</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;份数：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okorder">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>940</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>下单</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backtomain">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>940</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>810</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;请输入：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>930</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;用餐桌号：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="tableid"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
